translator.go

```diff
@@ -239,6 +240,63 @@ func blockTemplate(block *zjson.Block, blockKey string, key string, organization
 			},
 		},
 	}
+
+	if block.Action.Resources.CPU.Request != "" {
+		template.Container.Resources.Requests[corev1.ResourceCPU] = resource.MustParse(block.Action.Resources.CPU.Request)
```

Comment:
I believe MustParse will panic if it fails. We should regular Parse and check if err is nil.

Reply:
Fixed